wagtail/admin/templates/wagtailadmin/pages/index.html

```diff
@@ -43,9 +43,76 @@
     <script type="text/javascript">
         {% if ordering == 'ord' %}
             $(function() {
+                var currentlySelected;
+                var currentPosition;
+                var movedPageId;
+                var moveNTimesDirection = 0;
                 var reorderCount = 0;
                 var orderform = $('#page-reorder-form');
 
+                $("[data-orderer]").on("keydown", function(e) {
+                    let keyCodes = {
+                        enter: 13,
+                        upArrow: 38,
+                        downArrow: 40,
+                        escape: 27
+                    }
+
+                    if (currentlySelected) {
+                        // We want to prevent default key actions (like scrolling) when we have an object selected
+                        e.preventDefault();
+                    }
+
+                    var children = $('.listing tbody').children();
+
+                    let moveElement = function() {
+                        var index = currentPosition + moveNTimesDirection;
+                        if (index < 0) {
+                            index = 0
+                        } else if (index > children.length - 1) {
+                            index = children.length - 1
+                        }
+                        var url = "{% url 'wagtailadmin_pages:set_page_position' '999999' %}".replace('999999', currentlySelected.id.substring(5));
+                        url += `?position=${(index)}`;
+                        let CSRFToken = $('input[name="csrfmiddlewaretoken"]', orderform).val();
+                        $.post(url, {csrfmiddlewaretoken: CSRFToken}, function(){
+                            addMessage('success', `"${$(currentlySelected).data('page-title')}" has been moved successfully.`)
```

Comment:
* wagtail/admin/templates/wagtailadmin/base.html
* `<div class="messages" role="status">`
Setting role stats on the page messages will help generally with accessibility so that new messages that get added dynamically will update screen readers.

Have a read of https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/status_role